playground/Playground.tsx

```diff
@@ -1,14 +1,12 @@
 import * as React from 'react';
-import {Page, AppProvider} from '@shopify/polaris';
+import {Page} from '@shopify/polaris';
 
 interface State {}
 
 export default class Playground extends React.Component<never, State> {
   render() {
     return (
-      <AppProvider>
-        <Page title="Playground" />
-      </AppProvider>
+      <Page title="Playground">{/* Add the code you want to test here */}</Page>
```

Comment:
I assume prettier brings it onto 1 line.

Reply:
Yep due to the line length as you've saved a whole two chars of indentation.